ca/server.go

```diff
@@ -36,6 +36,10 @@ type Server struct {
 	joinTokens                  *api.JoinTokens
 	reconciliationRetryInterval time.Duration
 
+	// skipAutoUpdateRootCA, if set to true, causes the CA server to skip automatically updating
+	// the signing credentials based on changes in the memory store
+	skipAutoUpdateRootCA bool
```

Comment:
Another option I was considering was to have a separate startable goroutine on the server that would watch the memory store and auto-update the `RootCA`.  Then whomever is responsible for starting it would have to cancel it as well.  Would this be more preferable?  But again, this would only be used for the tests.